dienst2/models.py

```diff
@@ -0,0 +1,27 @@
+from django.db import models
+from django.contrib.auth.models import AbstractBaseUser
+
+class CHConnectUser(models.Model):
+    REQUIRED_FIELDS = []
+    USERNAME_FIELD = 'sub'
+
+    user_info = None
+    is_active = True
+    #sub = ''
```

Comment:
Commented code

Reply:
Actually this entire file was a remnant of an earlier experiment and can be removed. We do not currently have our own User model, and apparently it is very hard to migrate to a situation where we do, so instead we keep the default `User` model from `django.contrib.auth.models` and create/update that in `auth.py`.